starter-code/index.html

```diff
@@ -0,0 +1,227 @@
+            <div class="carousel-caption">
+                <h2> MEOW! </h2>
+                <p> ¡Es un gato pequeño!
+            </div>
+        </div>
+        <div class="item">
+            <img src="img/slider.jpg">
+            <div class="carousel-caption">
+                ...
+            </div>
+        </div>
+    </div>
+
+
+    <div class="mid-container">
+        <div class="row">
+            <div class="col-sm-4 widget">
+                <span class="glyphicon glyphicon-glass"></span>
+                <h3> Web design </h3>
+                <p> Bacon ipsum dolor amet tenderloin turducken filet mignon, kevin ham hock pastrami alcatra tongue spare ribs.
+                    Ham hock shoulder shank, brisket t-bone sirloin kielbasa short ribs beef ribs.
+                </p>
+            </div>
+
+
+            <div class="col-sm-4 widget">
+                <span class="glyphicon glyphicon-star"></span>
+                <h3> Web development </h3>
+                <p> Spare ribs short loin ham hock tail jowl ground round capicola prosciutto filet mignon boudin beef kielbasa
+                    venison pork flank. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nspendisse at erat
+                    luctus .</p>
+            </div>
+
+            <div class="col-sm-4 widget">
+                <span class="glyphicon glyphicon-heart"></span>
+                <h3> User interface </h3>
+                <p> Doner chicken meatball sausage, jerky kevin spare ribs. Tongue ribeye swine, biltong hamburger tri-tip ball
+                    tip chicken filet mignon beef t-bone short ribs bacon shoulder prosciutto. </p>
+            </div>
+        </div>
+
+        <div class="section-title">
+            <h2> Latest Projects </h2>
+            <p> Few of the latest portfolio items </p>
+        </div>
+
+
+        <div class="boxitems">
+
+            <div class="col-xs-6 col-md-3">
+                <img class='hthumb' src="img/project-1.jpg">
+                <p class='third'>Pancetta ground round bacon, shankle jowl flank cupim. </p>
+            </div>
+
+            <div class="col-xs-6 col-md-3">
+                <img class='hthumb' src="img/project-3.jpg">
+                <p class='third'>Doner cupim porchetta t-bone, kielbasa landjaeger boudin.</p>
+            </div>
+
+            <div class="col-xs-6 col-md-3 ">
+                <img class='hthumb' src="img/project-4.jpg">
+                <p class='third'>Li Europan lingues es membres del sam familie. Lor separat existentie es un myth..</p>
+            </div>
+
+            <div class="col-xs-6 col-md-3">
+                <img class='hthumb' src="img/project-2.jpg">
+                <p class='third'>Lor separat existentie es un myth..</p>
+            </div>
+        </div>
+
+
+
+        <div class="section-title">
+            <h2> Latest Articles </h2>
+            <p> Latest posts from the Blog </p>
+        </div>
+
+        <div class="third-boxitems">
```

Comment:
You can use more things apart divs, like sections and articles. Remember: semantic HTML.